lab-said/__test__/server.test.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const server = require('../server.js');
+const superagent = require('superagent');
+
+describe('Testing the server file', function () {
+  afterAll((done) => {
+    server.close(done);
+  });
+
+  describe('POST method, / endpoint', () => {
+
+    test('should return a status code of 200', done => {
+      superagent.post('localhost:3000/')
+      .send({})
+      .set('Content-Type', 'application/json')
+      .end((err, res) => {
+        expect(err).toBeNull();
```

Comment:
This test is failing. It is most likely caused because you don't have a '/' route set up. This is also preventing any of your other tests from running.